go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/cloudevents/sdk-go/v2 v2.15.2
 	github.com/confluentinc/confluent-kafka-go/v2 v2.4.0
 	github.com/deepmap/oapi-codegen/v2 v2.1.0
-	github.com/getkin/kin-openapi v0.124.0
+	github.com/getkin/kin-openapi v0.123.0
```

Comment:
Dependabot will keep updating this. Also, due to how dependencies work in Go, any number of dependency updates with a common dependency could get this upgraded again, too.

Make sure to add a `replace` that pins the version.